perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -513,19 +543,50 @@ def _extract_vm_metrics(time_intervals_list, folders, mount_type) -> list:
 
     if((testing_folder.name == '1KB_100000files_0subdir' or testing_folder.name == '1KB_200000files_0subdir') and mount_type == 'gcs_bucket'):
```

Comment:
Create a constant list in this script (VM_METRICS_EXTRACTION_DIRS = [dir1, dir2]) and check if testing_folder.name in VM_METRICS_EXTRACTION_DIRS here.